addons/mail/static/tests/messaging/messaging.test.js

```diff
@@ -97,7 +101,7 @@ test("Show conversations with new message in chat hub (outside of discuss app)",
             thread_model: "discuss.channel",
         })
     );
-    await click(".o-mail-DiscussSidebar-item:contains('Dumbledore'):has(.badge:contains(1))");
+    await click(".o-mail-DiscussSidebar-item:contains('Dumbledore'):has(.badge:contains(2))");
```

Comment:
Ok I see, the counter is 2 because the user interactions above (close chat window) doesn't mark as read.

I think for the sake of test it's best to mark as read. The assertions that the test makes in the beginning exist for the sole purpose to have positive assertions of selectors, so that we can use reliably use the negative assertions.

IMO marking the conversation as read and keeping counter 1 makes it more natural to read the test, but maybe that's more personal preference.